docs/standard/frameworks.md

```diff
@@ -14,139 +14,150 @@ ms.assetid: 6ef56a2e-593d-497b-925a-1e25bb6df2e6
-|                            |              | sl5                                          |
-| Windows Phone              | wp           | wp [wp7]                                     |
-|                            |              | wp7                                          |
-|                            |              | wp75                                         |
-|                            |              | wp8                                          |
-|                            |              | wp81                                         |
-|                            |              | wpa81                                        |
-| Universal Windows Platform | uap          | uap [uap10.0]                                |
-|                            |              | uap10.0 [win10] [netcore50]                  |
-
-## Deprecated frameworks
-
-The following frameworks are deprecated. Packages targeting these frameworks should migrate to the indicated replacements.
-
-| Deprecated framework | Replacement |
-| -------------------- | ----------- |
-| aspnet50             | netcoreapp  |
-| aspnetcore50         |             |
-| dnxcore50            |             |
-| dnx                  |             |
-| dnx45                |             |
-| dnx451               |             |
-| dnx452               |             |
-| dotnet               | netstandard |
-| dotnet50             |             |
-| dotnet51             |             |
-| dotnet52             |             |
-| dotnet53             |             |
-| dotnet54             |             |
-| dotnet55             |             |
-| dotnet56             |             |
-| netcore50            | uap10.0     |
-| win                  | netcore45   |
-| win8                 | netcore45   |
-| win81                | netcore451  |
-| win10                | uap10.0     |
-| winrt                | netcore45   |
+When you target one or more frameworks in an app or library, you're specifying the sets and versions of APIs that you'd like to make available to the app or library. You specify target frameworks in the project file using Target Framework Monikers (TFMs). When you specify multiple target frameworks, you may conditionally reference assemblies for each target framework. In your code, you can conditionally compile against those assemblies by using preprocessor symbols with *if-then-else* logic.
 
-## Precedence
+An app or library can target a version of [.NET Standard](~/docs/standard/net-standard.md). .NET Standard versions represent standardized sets of APIs across all .NET frameworks. For example, a library can target .NET Standard 1.6 and gain access to APIs that function across several .NET implementations all at once.
+
+An app or library can also target a specific .NET implementation to gain access to implementation-specific APIs. For example, an app that targets Xamarin.iOS gets access to Xamarin-provided iOS API wrappers.
```

Comment:
I don't mind a little Xamarin 💟, but it might be even worse than what you point out. There is *NO* `Xamarin.iOS` TFM in this topic! ... *That's not just a little bit confusing imo.* When u say "Windows OS API" wrappers, do you mean something like `uap10.0` for the example? What TFM did u have in mind for the example?

Let's patch-link to your .NET Glossary after your topic appears. Merge conflicts make me sad. :cry: I've made a note-to-self to set it up. Also, the "2.0" language should prob wait. This topic can merge ASAP before 2.0 ships in its current form.